### PR TITLE
Adds brazilian portuguese language support

### PR DESCRIPTION
**Pull request by labmorales**
Don't know if it works on every version of redmine. Tested in redmine 3.3.8.stable. But i think that translation files work on every version.

I'm looking to contributed to fix some bugs, there is a easy way to test my code in several version o redmine?

### CODE REVIEW

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
+en:
```
Comment (arBmind) [inline config/locales/pt-BR.yml]: This should be `br` ?

**Comment by arBmind** [comment]
Travis-CI will run on different versions of Ruby and Redmine.
You might want to enable it for your fork.

File: config/locales/pt-BR.yml (hunk 1 above)
Comment (labmorales) [reply config/locales/pt-BR.yml]: Should be pt-BR. Sorry about that. I'll commit a fix shortly.

Comment (labmorales) [reply config/locales/pt-BR.yml]: Fixed. But some tests are not passing in ruby 2.3.4 (Travis CI). The issue does not seem to be related to the translation file though. Should i try to fix it?

**Comment by arBmind** [comment]
> …some tests are not passing in ruby 2.3.4 (Travis CI). The issue does not seem to be related to the translation file though. Should i try to fix it?

No, this seems fixed in the next_version branch.

**Pull request merged**